addons/point_of_sale/static/src/js/models.js

```diff
@@ -461,6 +462,33 @@ class PosGlobalState extends PosModel {
         });
         this._loadProductProduct(products);
     }
+    // load the partners based on the ids
+    async _loadPartners(partnerIds) {
+        if (partnerIds.length > 0) {
+            var fields = _.find(this.models, function(model){ return model.label === 'load_partners'; }).fields;
+            var domain = [['id','in', partnerIds]];
+            const fetchedPartners = await this.env.services.rpc({
```

Comment:
You need to use the method from pos.session: get_pos_ui_res_partner_by_params to fetch the partners.

Reply:
Thanks for your feedback.
It's done, and I tested it.